### ISSUE TITLE
Password protected wallet and transaction send

### ISSUE THREAD

**Issue by etherwvlf** [zingolabs/zingo-pc#208]
It would be great if we had password protected and encrypted wallet files and also a password protected send transaction button to protect from mistakes.
Maybe using some elliptic curve or post quantum encryption algo.

**Comment by juanky201271** [comment]
Thanks for your suggestion @etherwvlf , in fact @Edicksonjga said several times the same. We are considering to implement this soon, but we are concern as well about the user in some point is going to forget the password and lost the access to the wallet file...

**Comment by etherwvlf** [comment]
Well, nothing in this world is foolproof.
People should thread carefully, it's their own funds at stake after all.
Maybe this will help make them more vigilant.

**Comment by juanky201271** [comment]
I agree, of course, but we are always thinking about our basic users, but I have to say a Zcash Desktop App needs someone kind of medium-advance level... @etherwvlf thanks though.

**Comment by juanky201271** [comment]
Hello @etherwvlf We were working on this issue a bit... We included an opt-in to activate the device security in the App, in two points: launching the App & send funds. We also start working on the opt-in of encrypt the wallet file (zingolib) with a passphrase to prevent loosing the funds if the attacker copy the .dat file.  https://github.com/zingolabs/zingolib/issues/2330

cc: @Edicksonjga

**Comment by etherwvlf** [comment]
Nice.
Thank you kindly @juanky201271 !
I've loved everything that you've done with this fine software up till now and this would only complement it further :)
